### PR TITLE
Update web-report-designer.md

### PR DESCRIPTION
**Pull request by elkinmilen**

### CODE REVIEW

File: user-guide/web-report-designer.md [hunk 1]
```diff
@@ -11,28 +11,36 @@ position: 200
...
 
-![wrd_reportView](../images/wrd_reportView.png)
+![Launch Web report Designer](../images/wrd_reportView.png)
 
 ### Data Connectivity
 
 Report Server supports [WebService](https://docs.telerik.com/reporting/webservicedatasource-component), [JSON](https://docs.telerik.com/reporting/jsondatasource-component), [SQL](https://docs.telerik.com/reporting/sqldatasource) and inline [CSV](https://docs.telerik.com/reporting/csvdatasource-component) data sources.
 
+### Report Resources
+The Web report Designer allows uploading shared resources like images, styles, and data on the server.
```
Comment (nelci592) [inline user-guide/web-report-designer.md:33]: Report *

File: user-guide/web-report-designer.md [hunk 2]
```diff
@@ -11,28 +11,36 @@ position: 200
...
+It makes them easily accessible and allows to reuse them among different reports to achieve and maintain a common report vision.
+It also makes it easy to swap the offline data for multiple reports visualizing it.
```
Comment (nelci592) [inline user-guide/web-report-designer.md:35]: offline data in JSON or CSV format ?

**Review by nelci592 — APPROVED** [review]

File: user-guide/web-report-designer.md (hunk 1 above)
Comment (elkinmilen) [reply user-guide/web-report-designer.md:33]: Changed, thanks!

File: user-guide/web-report-designer.md (hunk 2 above)
Comment (elkinmilen) [reply user-guide/web-report-designer.md:35]: Added, thanks!

**Pull request merged**